packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -578,12 +578,13 @@ export class AccountHandler {
     this.log.begin('Account.finishSetup', request);
     const form = request.payload as any;
     const authPW = form.authPW;
+    let uid;
```

Comment:
Is this additional variable necessary? Could you just use `payload.uid` in the if statement?

Reply:
yes because in order to get the payload you need to call `jwt.verify` so if for whatever reason that call were to throw an error (its within the `try` block) we dont want to attempt to drop their reminder. similarly if we did want to remove their reminder we dont want `jwt.verify` to potentially return the payload in the `try` block but not in the `catch` block and then not drop their reminder (if that all made sense). i also looked into how to rewrite this but ran up against the TS compiler. basically i wanted `uid` to be a shared variable between the try and catch block so if its set in one, its set in both.

Reply:
Yikes, I somehow missed the try/catch. Sorry about that.